VoiceControl/CHANGELOG.md

```diff
@@ -5,6 +5,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.0.0] - 2022-08-18
```

Comment:
As discussed, this includes a breaking change. Pls first make a minor version update to deprecate the voiceSessionByText API to ensure existing clients are not affected. Details about deprecation at under https://github.com/rdkcentral/rdkservices#versioning.